pkgs/development/python-modules/cytoolz/default.nix

```diff
@@ -16,6 +17,15 @@ buildPythonPackage rec {
     sha256 = "84cc06fa40aa310f2df79dd440fc5f84c3e20f01f9f7783fc9c38d0a11ba00e5";
   };
 
+  patches = [
+    # temporal fix for a test failure: https://github.com/pytoolz/cytoolz/issues/122
+    (fetchpatch {
+      name = "py37.patch";
+      url = "https://salsa.debian.org/python-team/modules/python-cytoolz/raw/master/debian/patches/py37.patch";
```

Comment:
```suggestion
      url = "https://salsa.debian.org/python-team/modules/python-cytoolz/raw/5ce4158deefc47475d1e76813f900e6c72ddcc6e/debian/patches/py37.patch";
```

Reply:
You should also change the URL in the commit messsage.
And a minor nitpick: some people prefer using quotes-free URLs given this is a thing in Nix.